packages/client/hmi-client/src/page/data-explorer/components/tera-asset-card.vue

```diff
@@ -201,7 +201,9 @@ const title = computed(() => {
 	let value = '';
 	if (props.resourceType === ResourceType.XDD) {
 		value =
-			props.source === DocumentSource.XDD ? (props.asset as Document).title : (props.asset as DocumentAsset).name ?? '';
+			props.source === DocumentSource.XDD
+				? (props.asset as Document).title
+				: ((props.asset as DocumentAsset).name ?? '');
```

Comment:
I don't know what happened here, the formatter has a mind of its own

Reply:
Nope, I updated Prettier from `3.3.2` to `3.3.3` and that is what happened...